changelog/7348.feature.rst

```diff
@@ -0,0 +1 @@
+Add an example of plotting a rectangle on a map with a rotation angle relative to the axes (:ref:`sphx_glr_generated_gallery_plotting_plot_rotated_rectangle.py).`
```

Comment:
```suggestion
Add an example of plotting a rectangle on a map with a rotation angle relative to the axes (:ref:`sphx_glr_generated_gallery_plotting_plot_rotated_rectangle.py`).
```
Should fix:
`<unknown>:14: WARNING: undefined label: 'sphx_glr_generated_gallery_plotting_plot_rotated_rectangle.py).'